scan/src/reusable/Section.re

```diff
@@ -0,0 +1,14 @@
+module Styles = {
+  open Css;
+
+  let base = (~pt, ~pb, ()) => style([paddingTop(`px(pt)), paddingBottom(`px(pb))]);
+
+  let bgColor = color => style([backgroundColor(color)]);
+};
+
+[@react.component]
+let make = (~children, ~pt=0, ~pb=0, ~bg=Colors.white) => {
```

Comment:
As I commented above about the section top spacing, I think we should put default paddingTop and paddingBottom to `24px`